src/components/Grid/Grid.tsx

```diff
@@ -38,11 +37,28 @@ export interface Props {
     | '1-2-1up';
 }
 
+export interface GridItemProps {
+  /**
+   * Child node(s) that can be nested inside component
+   */
+  children?: ReactNode;
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+}
+
 /**
+ * The Grid component is deprecated and will be removed in an upcoming release.
+ * Instead, please make use of utility libraries, like Tailwind CSS:
+ * * https://tailwindcss.com/docs/display#grid
```

Comment:
Linking to TW docs upon deprecation, as we want to focus EDS on presentation layer, not page layout layer. (we can document guidance around page layout in zeroheight, however, and include code examples using some code sandbox software like codepen, etc.)